enterprise/api/filters.py

```diff
@@ -128,17 +127,23 @@ def filter_queryset(self, request, queryset, view):
 
 class EnterpriseCustomerInviteKeyFilterBackend(filters.BaseFilterBackend):
     """
-    Filter backend to return invite keys under the user's enterprise only.
+    Filter backend to return invite keys under the user's enterprise(s) only.
+    Supports filtering by enterprise_customer_uuid.
 
     * Staff users will bypass this filter.
     """
 
     def filter_queryset(self, request, queryset, view):
-        """
-        Filter out keys that are not part of the user's enterprise.
-        """
+        query_params = {}
+
         if not request.user.is_staff:
-            enterprise_customer_uuid = get_enterprise_customer_from_user_id(request.user.id)
-            queryset = queryset.filter(enterprise_customer_id=enterprise_customer_uuid)
+            user_enterprise_customer_uuids = [
+                ec.enterprise_customer_id for ec in EnterpriseCustomerUser.objects.filter(user_id=request.user.id)
+            ]
+            query_params.update(enterprise_customer_id__in=user_enterprise_customer_uuids)
 
-        return queryset
+        enterprise_customer_uuid = request.query_params.get('enterprise_customer_uuid', None)
+        if enterprise_customer_uuid:
+            query_params.update(enterprise_customer_id=enterprise_customer_uuid)
```

Comment:
`enterprise_customer_id=enterprise_customer_uuid`
How hard would it be for us to fix the naming mismatch here? How deep does this run?

Reply:
Django automatically populates the foreign key column name by appending _id. We could use db_column='enterprise_customer_uuid' to specify a name but we don't do it for any of the foreign keys we have. Alternatively query_params.update(enterprise_customer=enterprise_customer_uuid) should also work here.

Reply:
IIRC @long74100 and I were chatting about these mismatches the other day. fwiw just did a quick search across the edX github org... 

uses of `enterprise_id` in code: 104
uses of `enterprise_uuid` in code:  84
uses of `enterprise_customer_id` in code: 113
uses of `enterprise_customer_uuid` in code:  476

Might be part of a broader tech debt initiative to add to our backlog? 🤔

Reply:
I can add this as a tech debt item. i would assume that our default should always be the UUID moniker?

Reply:
my vote is for uuid 🤔